config/config.exs

```diff
@@ -119,8 +119,27 @@ config :trento, Trento.Scheduler,
   debug_logging: false
 
 config :trento, Trento.Integration.Telemetry, adapter: Trento.Integration.Telemetry.Suse
-config :trento, Trento.Integration.Checks, adapter: Trento.Integration.Checks.Runner
-config :trento, Trento.Clusters, checks_adapter: Trento.Clusters.Runner
+
+config :trento, :messaging, adapter: Trento.Messaging.Adapters.AMQP
+config :trento, Trento.Integration.Checks, adapter: Trento.Integration.Checks.Wanda
```

Comment:
This adapter remains here as we might really likely create the new wanda mock "faker"